main/test/ca/uwaterloo/flix/library/TestList.flix

```diff
@@ -2894,86 +2894,86 @@ def distinctWith09(): Bool =
-
-@test
-def literal03(): Bool =
-    List#{1, 2, 3} == 1 :: 2 :: 3 :: Nil
-
-@test
-def literal04(): Bool =
-    List#{'a', 'b', 'c'} == 'a' :: 'b' :: 'c' :: Nil
-
-@test
-def literal05(): Bool =
-    List#{List#{1, 2}, List#{3}, List#{4, 5}} == (1 :: 2 :: Nil) :: (3 :: Nil) :: (4 :: 5 :: Nil) :: Nil
-
-@test
-def literal06(): Bool =
-    not List.isEmpty(List#{{}})
-
-/////////////////////////////////////////////////////////////////////////////
-// frequency                                                               //
-/////////////////////////////////////////////////////////////////////////////
-@Test
-def frequency01(): Bool =
-    let l = List#{"a"};
-    let m = Map#{"a" => 1};
-    Assert.eq(m, List.frequency(l))
-
-@Test
-def frequency02(): Bool =
-    let l = List#{"a", "b", "c", "a", "b", "a"};
-    let m = Map#{"a" => 3, "b" => 2, "c" => 1};
-    Assert.eq(m, List.frequency(l))
-
-@Test
-def frequency03(): Bool =
-    let l = List#{"b", "c", "a", "a", "b", "a", "c", "a"};
-    let m = Map#{"a" => 4, "b" => 2, "c" => 2};
-    Assert.eq(m, List.frequency(l))
-
-@Test
-def frequency04(): Bool =
-    let l = (List#{}:List[Int32]);
-    let m = Map#{};
-    Assert.eq(m, List.frequency(l))
+    /////////////////////////////////////////////////////////////////////////////
+    // Literals                                                                //
+    /////////////////////////////////////////////////////////////////////////////
+
+    @Test
+    def literal01(): Bool =
+        List#{} == (Nil: List[String])
+
+    @Test
+    def literal02(): Bool =
+        List#{1} == 1 :: Nil
+
+    @Test
+    def literal03(): Bool =
+        List#{1, 2, 3} == 1 :: 2 :: 3 :: Nil
+
+    @Test
+    def literal04(): Bool =
+        List#{'a', 'b', 'c'} == 'a' :: 'b' :: 'c' :: Nil
+
+    @Test
+    def literal05(): Bool =
+        List#{List#{1, 2}, List#{3}, List#{4, 5}} == (1 :: 2 :: Nil) :: (3 :: Nil) :: (4 :: 5 :: Nil) :: Nil
+
+    @Test
+    def literal06(): Bool =
+        not List.isEmpty(List#{{}})
+
+    /////////////////////////////////////////////////////////////////////////////
+    // frequency                                                               //
+    /////////////////////////////////////////////////////////////////////////////
+    @Test
+    def frequency01(): Bool =
+        let l = List#{"a"};
+        let m = Map#{"a" => 1};
+        Assert.eq(m, List.frequency(l))
+
+    @Test
+    def frequency02(): Bool =
+        let l = List#{"a", "b", "c", "a", "b", "a"};
+        let m = Map#{"a" => 3, "b" => 2, "c" => 1};
+        Assert.eq(m, List.frequency(l))
+
+    @Test
+    def frequency03(): Bool =
+        let l = List#{"b", "c", "a", "a", "b", "a", "c", "a"};
+        let m = Map#{"a" => 4, "b" => 2, "c" => 2};
+        Assert.eq(m, List.frequency(l))
+
+    @Test
+    def frequency04(): Bool =
+        let l = (List#{}:List[Int32]);
+        let m = Map#{};
+        Assert.eq(m, List.frequency(l))
 }
```

Comment:
add extra line break both before and after so its easier to see the module close